Avoid collision with C library using NO_PROTOTYPES found in code already

Patch by Robin Getz <rgetz@blackfin.uclinux.org>

--- a/dhry_1.c
+++ b/dhry_1.c
@@ -28,7 +28,9 @@
 int             Arr_1_Glob [50];
 int             Arr_2_Glob [50] [50];
 
+#ifndef NO_PROTOTYPES
 extern char     *malloc ();
+#endif
 Enumeration     Func_1 ();
   /* forward declaration necessary since Enumeration may not simply be int */
 
@@ -45,14 +45,18 @@
 
 #ifdef TIMES
 struct tms      time_info;
+#ifndef NO_PROTOTYPES
 extern  int     times ();
                 /* see library function "times" */
+#endif
 #define Too_Small_Time 120
                 /* Measurements should last at least about 2 seconds */
 #endif
 #ifdef TIME
+#ifndef NO_PROTOTYPES
 extern long     time();
                 /* see library function "time"  */
+#endif
 #define Too_Small_Time 2
                 /* Measurements should last at least 2 seconds */
 #endif
